### PR TITLE
Fix and upgrade WinUI to 2.7

### PR DESCRIPTION
**Pull request by wroyca**

### CODE REVIEW

File: Win32App/Win32App.cpp [hunk 1]
```diff
@@ -5,6 +5,9 @@
 #include "Win32App.h"
 #include "MrmHook.h"
 
+// https://github.com/ysc3839/SingleExeXamlIsland#winui-2
+#pragma comment(lib, "OneCoreUAP_apiset.Lib")
```
Comment (ysc3839) [inline Win32App/Win32App.cpp]: Why move library import here? Is there any issue with it?

Comment (wroyca) [reply Win32App/Win32App.cpp]: > Why move library import here? Is there any issue with it?

It wasn't linked properly when it was in the vcxproj, but I didn't investigate why and just moved it into a pragma to make MSVC happier. 

Edit: Adding it again solved the linking errors that MSVC was throwing yesterday. I still don't know why it did that, but it is what it is.

Comment (ysc3839) [reply Win32App/Win32App.cpp]: Sorry, it's my fault. I forget to add them to x86 and x64 platform config.
https://github.com/ysc3839/SingleExeXamlIsland/blob/a63324309ab8380ab7a840d050e6d97b9fbf6b6e/Win32App/Win32App.vcxproj#L135-L165

**Comment by ysc3839** [comment]
Merged, thanks!

**Pull request merged**